### PR TITLE
Add New CSS Handles in QuantityStepper

### PR DESCRIPTION
**Pull request by FabioWill**
#### What problem is this solving?

Customize the QuantityStepper buttons and input with CSS Handles.

#### How to test it?

[Workspace](https://fabiowill--dentalpartner.myvtex.com/)

#### Screenshots or example usage:

Before:
Screen Shot 2021-07-12 at 16 25 23

After:
Screen Shot 2021-07-12 at 16 25 35

#### How does this PR make you feel? [:link:](http://giphy.com/)

![https://giphy.com/gifs/memecandy-LmNwrBhejkK9EFP504]

### CODE REVIEW

**Comment by FabioWill** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [ ]  Patch (backwards-compatible bug fixes)
> * [x]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

File: docs/README.md [hunk 1]
```diff
@@ -352,6 +352,10 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 | `quantityInputContainer`             |
 | `quantityInputMobileContainer`       |
 | `quantitySelectorContainer`          |
+| `quantitySelectorWrap`               |
```
Comment (victorhmp) [inline docs/README.md]:
```suggestion
| `quantitySelectorWrapper`               |
```

File: react/components/QuantityStepper.tsx [hunk 2]
```diff
@@ -136,12 +146,15 @@ const QuantityStepper: VFC<Props> = ({
   const uniqueId = id ? `product-list-quantity-stepper-${id}` : undefined
 
   return (
-    <div className="flex">
+    <div className={classnames(handles.quantitySelectorWrap, 'flex')}>
```
Comment (victorhmp) [inline react/components/QuantityStepper.tsx]:
```suggestion
    <div className={`flex ${handles.quantitySelectorWrapper}`}>
```

File: react/components/QuantityStepper.tsx [hunk 3]
```diff
@@ -10,6 +11,13 @@ import { parseDisplayValue } from './utils'
 import useQuantitySelectorState from './useQuantitySelectorState'
 import styles from './QuantityStepper.css'
 
+const CSS_HANDLES = [
+  'quantitySelectorWrap',
```
Comment (victorhmp) [inline react/components/QuantityStepper.tsx]:
```suggestion
  'quantitySelectorWrapper',
```

**Review by victorhmp — CHANGES_REQUESTED** [review]
Thanks for the PR @FabioWill! Left a few suggestions :)
Also, please update the CHANGELOG file.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.30.0] - 2021-07-20
+
+### Added
+- CSS Handles in QuantityStepper
+
```
Comment (victorhmp) [inline CHANGELOG.md]:
```suggestion
## [Unreleased]
### Added
- CSS Handles in QuantityStepper

```

**Review by victorhmp — APPROVED** [review]
LGTM, but please update the CHANGELOG with the suggestion I left :)

**Review by icazevedo — APPROVED** [review]

**Comment by GonzaloVicens** [comment]
Can someone provide information about when this modifications are going to be merged? We are in need of these CSS Handles for a company project. 
Thanks!

**Pull request merged**